src/javascript/app/ng-wig/views/ng-wig.html

```diff
@@ -1,10 +1,9 @@
 <div class="ng-wig">
   <ul class="nw-toolbar">
     <li class="nw-toolbar__item">
-      <button type="button" class="nw-button nw-button--header-one" title="Header" ng-click="execCommand('formatblock', '<h1>')"></button>
-    </li><!--
-    --><li class="nw-toolbar__item">
-      <button type="button" class="nw-button nw-button--paragraph" title="Paragraph" ng-click="execCommand('formatblock', '<p>')"></button>
+      <select class="form-control" ng-model="style" ng-change="execCommand('formatblock', style.value)" ng-options="style.name for style in styles">
+        <option ng-selected="true" disabled value="">--- Choose a style ---</option>
```

Comment:
I would suggest calling "Paragraph" "Normal text" and remove "Choose" option at all. (See, e.g., Google Docs)